store/tikv/coprocessor.go

```diff
@@ -598,10 +591,17 @@ func (it *copIterator) handleTaskOnce(bo *Backoffer, task *copTask, ch chan copR
 		// The resp is a stream and killed by cancel operation immediately.
 		timeout = 0
 	}
+	startTime := time.Now()
 	resp, err := sender.SendReq(bo, req, task.region, timeout)
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
+	costTime := time.Since(startTime)
+	if costTime > minLogCopTaskTime {
+		log.Infof("[TIME_COP_TASK] %s%s %s", costTime, bo, task)
+	}
+	metrics.TiKVCoprocessorCounter.WithLabelValues("handle_task").Inc()
```

Comment:
Do we need to update the label name of `handle_task`?

Reply:
I think it's ok to use the old name.